### PR TITLE
Update workarounds

### PR DESCRIPTION
**Pull request by russellkan**
Context: Many of the workarounds listed in the [README](https://github.com/innolitics/dicom-standard?tab=readme-ov-file#dicom-standard-workarounds) are out of date and have been fixed. The parsed files have not been updated for a while either since the `update-standard` action has been failing.

Changes:
- Remove outdated/fixed standard workarounds
- Add new workarounds as of rev2024a
- Update tests and README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,26 +236,36 @@ Certain parts of the DICOM Standard site cause errors when running the parser, o
 
 When we find one of these issues, we add a hard-coded fix in the relevant file and add a comment starting with 'Standard workaround' that describes the issue and links to its location in the Standard. To be aware when these fixes are obsolete, we add a unit test that fails once the issue no longer exists.
 
-Current standard workarounds (as of rev.2020b):
```
Comment (johndgiese) [inline README.md:239]: 👍

File: README.md [hunk 2]
```diff
@@ -236,26 +236,36 @@ Certain parts of the DICOM Standard site cause errors when running the parser, o
...
+| The [Confocal Microscopy Image Functional Group Macros](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.90.html#sect_A.90.1.5) section doesn't have a description, causing the "module_type" value to be None when it should be "Multi-frame" | `extract_ciod_func_group_macro_tables.py` |
+| The [Confocal Microscopy Tiled Pyramidal Image Functional Group Macros](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.90.2.5.html#sect_A.90.2.5) section doesn't have a description, causing the "module_type" value to be None when it should be "Multi-frame" | `extract_ciod_func_group_macro_tables.py` |
 
 (\*) This issue is not caused by a typo or error in the Standard but rather an exception from the normal format and thus does not have a unit test for a fix.
 
+Fixed workarounds:
```
Comment (johndgiese) [inline README.md:253]: Good thinking adding a list of fixed workarounds!

File: dicom_standard/extract_ciod_func_group_macro_tables.py [hunk 3]
```diff
@@ -22,18 +22,18 @@
...
 COLUMN_TITLES = ['macro', 'section', 'usage']
 
 
-# Standard workaround: Add missing "Image" to title of Table A.52.4.3-1
-# http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.52.4.3.html#table_A.52.4.3-1
 def clean_macro_table_name(table_name: str) -> str:
     clean_name = pl.clean_table_name(table_name)
-    if clean_name == 'Ophthalmic Tomography':
-        clean_name = 'Ophthalmic Tomography Image'
+    # if clean_name == 'Ophthalmic Tomography':
```
Comment (johndgiese) [inline dicom_standard/extract_ciod_func_group_macro_tables.py:31]: Dead code?

**Review by johndgiese — APPROVED** [review]
Nice work Russell!

File: dicom_standard/extract_ciod_func_group_macro_tables.py (hunk 3 above)
Comment (russellkan) [reply dicom_standard/extract_ciod_func_group_macro_tables.py:31]: Oh, nice catch. I had initially intended on leaving fixed workarounds as comments but as you saw, I ended up just adding a section to the README.

**Pull request merged**